File: psuedocode/activities/problem_7/color_set.cc
#include <iostream>
#include <set>
#include <string>
#include <vector>

using namespace std;

set<string> color_set(vector<string>& V){
  set<string> S;

  for (string i : V){
    if (i != "Blue"){
      S.insert(i);
    }
  }

  return S;
}

int main(){

  string s;
  vector <string> V;

  while(cin >> s) V.push_back(s);

  for (string i : color_set(V)){
    cout << i << endl;
  }

return 0;
}

File: psuedocode/activities/problem_7/color_set.py
import sys

def color_set(L):
    S = set() 

    for i in L:
        if i != "Blue":
            S.add(i)
    return sorted(S)


L = []
s = sys.stdin.readline().strip()

while s != "":
    L.append(s)
    s = sys.stdin.readline().strip()

for i in color_set(L):
    print(i)
